lib/paper_trail/events/destroy.rb

```diff
@@ -22,13 +22,18 @@ def data
           data[:object] = recordable_object(false)
         end
         if record_object_changes?
-          # Rails' implementation returns nothing on destroy :/
-          changes = @record.attributes.map { |attr, value| [attr, [value, nil]] }.to_h
-          data[:object_changes] = prepare_object_changes(changes)
+          data[:object_changes] = prepare_object_changes(notable_changes)
         end
         merge_item_subtype_into(data)
         merge_metadata_into(data)
       end
+
+      private
+
+      # Rails' implementation returns nothing on destroy :/
```

Comment:
Nice use of polymorphism!

When overriding methods in the future, please add a `# @override` comment, like this:

```
      # Rails' implementation returns nothing on destroy :/
      #
      # @override
```